module/VuFind/src/VuFind/Auth/LoginTokenManager.php

```diff
@@ -212,6 +227,20 @@ public function tokenLogin(string $sessionId): ?\VuFind\Db\Row\User
         return $user;
     }
 
+    /**
+     * Rotate login token in given series or create a new series
```

Comment:
Is this comment accurate for this method? If this is also rotating, does it make sense to have a separate method name for the public version?

Reply:
Oops. Revised.